cyclonedx_py/parser/poetry.py

```diff
@@ -29,10 +32,17 @@
 from ._debug import DebugMessageCallback, quiet
 
 
+class OmitCategory(str, Enum):
```

Comment:
Annotate with `@enum.unique` ?

Reply:
woudl be better, but we could add this in a refactoring/review later

Reply:
and this is pobably not `OmitCategory(str, Enum)` but `OmitCategory(Enum)`.

Reply:
i will check this after the merge. no worries.